packages/driver/cypress/integration/commands/actions/type_spec.js

```diff
@@ -604,6 +604,94 @@ describe('src/cy/commands/actions/type - #type', () => {
     })
   })
 
+  describe(`type(' ') fires click event on button-like elements`, () => {
+    beforeEach(() => {
+      cy.visit('fixtures/click-event-by-type.html')
+    })
+
+    const targets = [
+      'button-tag',
+      'input-button',
+      'input-image',
+      'input-reset',
+      'input-submit',
+    ]
+
+    describe(`triggers with single ' '`, () => {
```

Comment:
nit: using keyword space would make it easier to locate this test in the future.

Reply:
done.